src/layouts/CardsList/__tests__/CardsList.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react';
+import CardsList from '../CardsList';
+import Card from '../../Card/Card';
+
+describe('Test displaying flashcards', () => {
+    it('should contain at least one child element', () => {
+        
+        const mockCards = [
+            { _id: "1", front: "Marko", back: "Polo" },
+            { _id: "1", front: "Marko", back: "Polo" },
+            { _id: "1", front: "Marko", back: "Polo" },
+        ];
+        
+        render(
+            <CardsList>
```

Comment:
**issue:** This is not correct integration test, as you should operate on internal react state. You should mock get request here

Reply:
Can you check my test again?